collector/src/bin/collector.rs

```diff
@@ -628,6 +629,18 @@ enum Commands {
         #[command(flatten)]
         db: DbOption,
     },
+
+    /// Displays diff between two local bench result.
+    BenchCmp {
+        #[command(flatten)]
+        db: DbOption,
+
+        #[arg(long)]
+        base: String,
+
+        #[arg(long)]
```

Comment:
It looks like `clap` doesn't support an argument to be both positional and long at once :( I probably mistook it with `typer`/`click` from Python. So let's remove the `long` `--` form, it would be annoying, the default usage should be `bench_cmp a b`.